doc/nrf/libraries/modem/location.rst

```diff
@@ -152,6 +156,22 @@ The following options are related to the HERE service and can usually have the d
 * :kconfig:option:`CONFIG_LOCATION_SERVICE_HERE_HOSTNAME`
 * :kconfig:option:`CONFIG_LOCATION_SERVICE_HERE_TLS_SEC_TAG`
 
+The following options control the default location request configurations and are applied
+when :c:func:`location_config_defaults_set` function is called:
+
+* :kconfig:option:`CONFIG_LOCATION_REQUEST_DEFAULT_METHOD_FIRST` - Choice symbol for first priority location method.
+* :kconfig:option:`CONFIG_LOCATION_REQUEST_DEFAULT_METHOD_SECOND` - Choice symbol for second priority location method.
+* :kconfig:option:`CONFIG_LOCATION_REQUEST_DEFAULT_METHOD_THIRD` - Choice symbol for third priority location method.
+* :kconfig:option:`CONFIG_LOCATION_REQUEST_DEFAULT_INTERVAL`
+* :kconfig:option:`CONFIG_LOCATION_REQUEST_DEFAULT_TIMEOUT`
+* :kconfig:option:`CONFIG_LOCATION_REQUEST_DEFAULT_GNSS_TIMEOUT`
+* :kconfig:option:`CONFIG_LOCATION_REQUEST_DEFAULT_GNSS_ACCURACY`
+* :kconfig:option:`CONFIG_LOCATION_REQUEST_DEFAULT_GNSS_NUM_CONSECUTIVE_FIXES`
+* :kconfig:option:`CONFIG_LOCATION_REQUEST_DEFAULT_GNSS_VISIBILITY_DETECTION`
+* :kconfig:option:`CONFIG_LOCATION_REQUEST_DEFAULT_GNSS_PRIORITY_MODE`
+* :kconfig:option:`CONFIG_LOCATION_REQUEST_DEFAULT_CELLULAR_TIMEOUT`
+* :kconfig:option:`CONFIG_LOCATION_REQUEST_DEFAULT_WIFI_TIMEOUT`
```

Comment:
Method specific timeouts are not having impact with "ext architecture" and as discussed that should be mentioned.

Reply:
I've added comments into the config structures in location.h in PR https://github.com/nrfconnect/sdk-nrf/pull/9804.
Please check them.